### PR TITLE
fixed indentation issue with heading

### PR DESCRIPTION
**Pull request by ereli-cb**
"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

- Address formatting issue shown below
Screen Shot 2021-07-07 at 17 51 30

### CODE REVIEW

**Comment by julz** [comment]
Hi @ereli-cb - thanks for the contribution! Unfortunately this looks like it actually breaks the formatting :(. FWIW you can test changes locally following the [Contributor Guide here](https://knative.dev/development/help/contributor/mkdocs-contributor-guide/).

**Comment by ereli-cb** [comment]
@julz there was more than one place to fix, I think I got all of them in this file.
Let me figure out who needs to approve the CLA on our end.

File: docs/serving/samples/hello-world/helloworld-python/README.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ cd knative-docs/docs/serving/samples/hello-world/helloworld-python
    > Note: If your URL includes `example.com` then consult the setup instructions for
    > configuring DNS (e.g. with `sslip.io`), or [using a Custom Domain](../serving/using-a-custom-domain).
 
-   === "kubectl"
+=== "kubectl"
 
        ```bash
        kubectl get ksvc helloworld-python  --output=custom-columns=NAME:.metadata.name,URL:.status.url
```
Comment (julz) [inline docs/serving/samples/hello-world/helloworld-python/README.md:183]: unfortunately outdenting this line means the rest of the indentation is wrong and breaks the numbering (the "Now you can make a request.." bullet below should be numbered 2). I think what you need to do is make sure the tab is indented by four spaces (the problem is it was only indented by 3).

```suggestion
    === "kubectl"

        ```bash
        kubectl get ksvc helloworld-python  --output=custom-columns=NAME:.metadata.name,URL:.status.url
        ```

```
(and add an extra space to the start of the line of the rest of the tab group to match. There should be four spaces before the `===` lines, and 8 spaces before all the others).

If it works correctly, it should look like this:

![image](https://user-images.githubusercontent.com/354013/124919600-55d27b00-dfee-11eb-9360-067252499183.png)

(Notice the tabs are indented nicely underneath the numbered bullets, and the bullet beginning "Now you can make a request" starts with the right number)

**Comment by ereli-cb** [comment]
@googlebot I signed it!

**Comment by ereli-cb** [comment]
@googlebot I consent.

**Comment by ereli-cb** [comment]
> All (the pull request submitter and all commit authors) CLAs are signed, **but** one or more commits were authored or co-authored by someone other than the pull request submitter.
> 
> We need to confirm that all authors are ok with their commits being contributed to this project. Please have them confirm that by leaving a comment that contains only `@googlebot I consent.` in this pull request.
> 
> _Note to project maintainer:_ There may be cases where the author cannot leave a comment, or the comment is not properly detected as consent. In those cases, you can manually confirm consent of the commit author(s), and set the `cla` label to `yes` (if enabled on your project).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fknative%2Fdocs%2Fpull%2F3972) for more info**.

@googlebot I consent

**Comment by ereli-cb** [comment]
@googlebot I consent.

**Comment by ereli-cb** [comment]
@googlebot I consent

**Comment by omerbensaadon** [comment]
@googlebot I consent.

**Comment by thisisnotapril** [comment]
@senthilnathan could you please consent to this PR merge for the CLA check? Thanks!

**Comment by senthilnathan** [comment]
@googlebot I consent.

**Comment by omerbensaadon** [comment]
numbering is still broken on this page but this is helpful! Thank you @ereli-cb, I promise most PRs don't take this long to merge. 

/lgtm 
/approve

**Pull request merged**